timeseries/src/autogluon/timeseries/utils/features.py

```diff
@@ -386,7 +391,11 @@ def __init__(
         self.covariate_metadata: CovariateMetadata = covariate_metadata
         self.prediction_length: int = prediction_length
 
-    def _transform_series(self, data: pd.Series, is_categorical: bool, **kwargs) -> TimeSeriesDataFrame:
+    def _transform_static_series(self, feature_data: pd.Series, is_categorical: bool) -> Any:
```

Comment:
What's this for?

Reply:
This is an abstract method that I forgot to add before, but is clearly necessary as `transform` depends on it.